internal/service/gateway/gateway.go

```diff
@@ -92,7 +95,47 @@ func ResourceGateway() *schema.Resource {
 				Type:        schema.TypeString,
 				Computed:    true,
 			},
+			"plan": {
+				Description: planDescription,
+				Computed:    true,
+				Optional:    true,
+				Type:        schema.TypeString,
+			},
+			"address": {
+				Description: addressesDescription,
+				Computed:    true,
+				Optional:    true,
+				Type:        schema.TypeSet,
```

Comment:
Could this be `schema.TypeMap`?

Reply:
Oops, sorry. It could not. :)

Reply:
Is the max limit of 1 about to change at some point? If not, we could have `ip_address` and `ip_address_name` fields instead of the `address`

Reply:
It's quite possible it will change at some point :)